examples/getstarted/src/index.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
    *
    * This gives you an opportunity to extend code.
    */
-  register({ strapi }) {},
+  register(app) {
+  },
```

Comment:
Sorry but this will keep me up at night
```suggestion
  register(app) {},
```